README.md

```diff
@@ -1,31 +1,36 @@
-# Open Container Distributable Image Specification
+# Open Container Initiative Image Format Specification
 
-The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
+The OCI Image Format project creates and maintains the software shipping container image format spec (OCI Image Format). The goal of this specification is to create interoperable tools for building, transporting, and preparing a container image to run.
 
-This OCI project is tasked with creating a software shipping container image format spec (OCI Image Format) with security and naming as components.
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output both a [filesystem serialization](serialization.md) and an [image manifest](manifest.md). The combination of one or more filesystem serializations and image manifest is called the "OCI Image".
```

Comment:
This specification defines how to create an OCI Image.
And OCI Image is a combination of one or more [filesystem serializations](serialization.md) and an [image manifest](manifest.md).